concoction-lib/src/main/java/info/mmpa/concoction/Concoction.java

```diff
@@ -63,6 +68,39 @@ public Concoction withMaxInputDirectoryDepth(int inputDepth) {
 		return this;
 	}
 
+	/**
```

Comment:
The changes were tied to things required by this GUI work, and there's some overlap so cherry-picking would be a pain since it'd require editing portions of commits